native~/Runtime/src/TextureLoader.cpp

```diff
@@ -38,7 +77,7 @@ TextureLoader::loadTexture(const CesiumGltf::ImageCesium& image) {
           GetUnsafeBufferPointerWithoutChecks(textureData));
 
   size_t textureLength = textureData.Length();
-  assert(textureLength >= image.pixelData.size());
+  //assert(textureLength >= image.pixelData.size());
```

Comment:
Why was this commented out?

Reply:
I know assert is working in debug build. and I found that for comressed format encoded texture in this case this line, textureLength is bigger than  image.pixelData.size(). if texture decompressed to RGBA32 is ok but for transcoded texture I think assert condition have to be changed. so I just commented out this assert. 
in runtime I test these code and there was no crash here.

Reply:
I know assert is working in debug build. and I found that for comressed format encoded texture in this case this line, textureLength is bigger than image.pixelData.size(). if texture decompressed to RGBA32 is ok but for transcoded texture I think assert condition have to be changed. so I just commented out this assert.
in runtime I test these code and there was no crash here.

Reply:
sorry my mistake I restore this assert code.